app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
```

Comment:
According to the checklist, it's recommended to use parentheses when importing multiple modules or classes. You should change the import statement to use parentheses instead of a backslash.